packit_service/models.py

```diff
@@ -154,7 +154,7 @@ class CoprBuild(Base):
     build_logs_url = Column(String)
     # datetime.utcnow instead of datetime.utcnow() because its an argument to the function
     # so it will run when the copr build is initiated, not when the table is made
-    build_submitted_time = Column(DateTime, nullable=False, default=datetime.utcnow)
+    build_submitted_time = Column(DateTime, default=datetime.utcnow)
```

Comment:
Sorry, I thought it was causing the issue so I removed it. Will revert this once I fix the error.

BTW, I just checked, adding any field to the copr_builds table in the postgres models.py file causes this error.

Example, adding `abc = Column(Integer)` inside the table in an unmodified file from master will cause this issue.

Reply:
no need to sorry :), i just thought it was good idea to have it not nullable

yes you are right you have to update db schema once you change db definition in code

@TomasTomecek 
what is exact procedure to perform db schema update?

Reply:
generate it using alembic: https://alembic.sqlalchemy.org/en/latest/autogenerate.html

as for nullable: the thing is that we already have this in production with data in the table: once you change schema, you need to figure out what to do with present entries: either make it nullable, set a default, or figure it out during schema upgrade

Reply:
Thanks. 
My current local setup is docker based, so I make changes in packit-service files and then run docker-compose up with --build to test those changes.

Now running autogenerate probably changes some schema permanently, but where do I run it. I can't run it in a regular venv because alembic just says  No module named 'packit_service'.

Reply:
autogenerate only creates the schema files which are then meant to be applied by the upgrade command

hence autogen is safe to run

> Is there a way I can run autogenerate in docker packit-service container and then copy the changed files over to the actual code folder ?

yes, that should do

I usually run all this directly on my laptop (no venv, no containers) by starting a psql container and then exposing env vars in shell so alembic knows how to connect